guiEditor/src/components/propertyTab/propertyGrids/gui/commonControlPropertyGridComponent.tsx

```diff
@@ -41,51 +42,102 @@ interface ICommonControlPropertyGridComponentProps {
     onPropertyChangedObservable?: Observable<PropertyChangedEvent>;
 }
 
+
+
 export class CommonControlPropertyGridComponent extends React.Component<ICommonControlPropertyGridComponentProps> {
     constructor(props: ICommonControlPropertyGridComponentProps) {
         super(props);
     }
 
+    updateAlignment(alignment: string, value: number) {
+        const control = this.props.control;
+        if (control.typeName == "TextBlock") {
+            (this.props.control as any)["text" + alignment.charAt(0).toUpperCase() + alignment.slice(1)] = value;

```

Comment:
can this fail?

Reply:
It can but the expectation is the it's only called in 1 place and has the right arguments. I set the function to private but I see where you're going. 

If the property does that exist is just silently fails. Suggestions?

Reply:
silently fails is fine, as long as no exception stops the render loop

Reply:
Got it! All good then.